### ISSUE TITLE
function without brackets for arguments isn't flagged

### ISSUE THREAD

**Issue by zspitzer** [cflint/CFLint#745]
the following invalid function definition isn't flagged for missing brackets

```
private void function test {
}
``

**Comment by ghedwards** [comment]
I get a parse error which seems about right ?

**Comment by ghedwards** [comment]
@zspitzer do you get something different? Or disagree?
